Build site

richpauloo opened the pull request:
Will merge once osf.io links are in place and pushed to this PR. Just staging for now. On develop, we can view the site to make sure all looks good. If everything checks out, we will merge develop into main, give it a version stamp, and point the github.io site to publish the main branch (right now it publishes develop).

Final to-do:

- [x] rich sends @ryanpeek wrds-data: one for each course actually
- [x] ryan publishes on osf.io and sends links to @richpauloo 
- [x] rich re-builds the site and pushes to this PR, and merges
- [ ] rich + ryan review the published site on develop
- [ ] rich + ryan merge develop into main, version it, and point the github.io site to the main branch
- [ ] rich emails the Board and Vyki
- [ ] rich + ryan celebrate 🎉

ryanpeek commented:
OSF links are to you via email.

richpauloo commented:
Okay, site is built and live on r4wrds.com! Still publishing from develop, so let's review today and then pull it into main and stamp it with a version. Then we can email the Board and Vyki and call this done!

ryanpeek commented:
On [Intro landing page](http://r4wrds.com/intro/) the button at the bottom to advance the next lesson is linking to project management, not install R/RStudio.

richpauloo commented:
Fixed in the latest staged PR. Keep em coming!

ryanpeek commented:
Cool...I'm going through intermediate now.

On Fri, Jul 2, 2021 at 9:51 AM Rich Pauloo ***@***.***> wrote:

> Fixed in the latest staged PR. Keep em coming!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ryanpeek/r4wrds/pull/72#issuecomment-873131043>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMX3MH2LA4KWVQQVGZ5MDTTVXVCHANCNFSM47VTNVIA>
> .
>


-- 




*"When we try to pick out anything by itself, we find it hitched to
everything else in the universe."John Muir (My First Summer in the Sierra,
1911)*
*-----------------------------------------------------*
Ryan Peek, PhD (he/him)
Aquatic Ecologist, Post-Doctoral Researcher
Center for Watershed Sciences, UC Davis
*ryanpeek.org <https://ryanpeek.org/>*
***@***.****
530.754.5351


*-----------------------------------------------------*

richpauloo commented:
I just fixed a few more bugs (mostly related to links) in the intro course, and fixed dropdown links in intermediate. 

Merging that into develop shortly.

ryanpeek commented:
Module 3 in Intro needs a `3` in the title (Project Management)

ryanpeek commented:
Drop down menu in the intermediate course doesn't have any text and links appear broken....but all the lessons look good and all the bottom of the page buttons work/are correctly linked.

richpauloo commented:
Nice! Glad to know we found the same issues. They should be fixed in the latest PR.

intro m3: https://r4wrds.com/intro/m_project_management.html
intermediate example (check dropdown): https://r4wrds.com/intermediate/m_simple_shiny.html

ryanpeek commented:
Actually now that I'm digging into this its a bit more odd than I expected. The module drop down appears/works fine from the home page, and from all the lessons *except* Module 1, Module 5-Module 8. If you are on the specific module, it still seems to show a module drop down that's empty. This may be a re-render issue, but just FYI.

ryanpeek commented:
Ok! I think if those things are fixed it looks good to go.

richpauloo commented:
Alright, let's call review done. Pulling develop into main now. 

Can you change the "publish from" branch from develop -> main in the repo settings? I'm not sure if I have permissions.

ryanpeek commented:
Will do!

On Fri, Jul 2, 2021 at 10:47 AM Rich Pauloo ***@***.***>
wrote:

> Alright, let's call review done. Pulling develop into main now.
>
> Can you change the "publish from" branch from develop -> main in the repo
> settings? I'm not sure if I have permissions.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ryanpeek/r4wrds/pull/72#issuecomment-873161607>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMX3MC37Z5EDAVXW2RNT5LTVX3SZANCNFSM47VTNVIA>
> .
>


-- 




*"When we try to pick out anything by itself, we find it hitched to
everything else in the universe."John Muir (My First Summer in the Sierra,
1911)*
*-----------------------------------------------------*
Ryan Peek, PhD (he/him)
Aquatic Ecologist, Post-Doctoral Researcher
Center for Watershed Sciences, UC Davis
*ryanpeek.org <https://ryanpeek.org/>*
***@***.****
530.754.5351


*-----------------------------------------------------*
